src/AccessesPostsLazily.php

```diff
@@ -67,7 +66,7 @@ trait AccessesPostsLazily
      *   // No additional overhead here.
      * }
      * ```
-     * @return \Timber\PostCollectionInterface The realized PostQuery.
+     * @return PostCollectionInterface The realized PostQuery.
```

Comment:
This throws a static analysis error. Is this something we must take care of? Not sure if this was something that we already discussed in an issue.

Reply:
@gchtr , mentioned this over here: https://github.com/timber/timber/pull/2668

Reply:
Haven't figured it out yet but this shouldn't prevent us from merging the PR. We don't require PHPStan to pass (yet). :)